src/SFA.DAS.ApprenticeAccounts.Api/Filters/SuccessfulDependencyFilter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace SFA.DAS.ApprenticeAccounts.Api.Filters;
+
+public class SuccessfulDependencyFilter : ITelemetryProcessor
+{
+    private ITelemetryProcessor Next { get; set; }
+
+    public SuccessfulDependencyFilter(ITelemetryProcessor next)
+    {
+        this.Next = next;
+    }
+
+    public void Process(ITelemetry item)
+    {
+        var statusesToIgnore = new List<HttpStatusCode>{ HttpStatusCode.NotFound, HttpStatusCode.BadRequest };
+
+        if (!OKtoSend(item)) { return; }
```

Comment:
We need to send telemetry for logging purpose, this will not send telemetry at all